packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -256,6 +264,7 @@ async function createOutgoingPayment(
     )
     const { debitAmount, incomingPayment } = options
     const quoteOrError = await deps.quoteService.create({
+      tenantId,
```

Comment:
do we need to pass in tenantId into the wallet address lookup `const walletAddress = await deps.walletAddressService.get(walletAddressId)`? Otherwise it seems possible to create an outgoing payment for a tenantId that is different than the wallet address's